locales/en/app.ftl

```diff
@@ -1,16 +1,18 @@
-# String IDs beginning with "-product" and "-brand" should remain in English.
-# They should not be:
-# - Declined to adapt to grammatical case.
-# - Transliterated.
-# - Translated.
+## The following messages are brand and should be kept entirely in English
+## unless otherwise indicated.
+
 -product-name = Firefox Monitor
 -product-name-nowrap = <span class="nowrap">{-product-name}</span>
 -product-short-name = Monitor
 -brand-name = Firefox
+-brand-firefox-browser = Firefox Browser
```

Comment:
This has been a rollercoaster. We're back to "Browser" being localizable, so we can remove this term.